src/current/cockroachcloud/serverless-cluster-management.md

```diff
@@ -83,8 +83,8 @@ To add regions to your cluster:
 
 1. Click **Add region**.
 1. Choose the region you want to add or use the suggested one.
-1. In the **Summary** sidebar, verify the hourly estimated cost for the cluster.
-1. Click **Update**.
+1. In the **Configuration** sidebar, verify the hourly estimated cost for the cluster.
```

Comment:
I think this is the only place referring to the sidebar as "Configuration" sidebar. Should we just say "sidebar" everywhere?